docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -275,156 +275,218 @@ RAJA policies for GPU execution using CUDA or HIP are essentially identical.
-            different thread dimensions), the product of sizes of the
-            corresponding iteration spaces cannot be greater than the
-            maximum allowable threads per block. Typically, this is
-            1024 threads per block. Attempting to execute a kernel with more
-            than the maximum allowed the CUDA runtime
-            to complain about *illegal launch parameters.*
+ ========================================= ============= =======================================
+ CUDA/HIP Execution Policies               Works with    Brief description
+ ========================================= ============= =======================================
+ cuda/hip_exec<BLOCK_SIZE>                 forall,       Execute loop iterations
+                                           scan,         in a GPU kernel launched
+                                           sort          with given thread-block
+                                                         size. Note that the
+                                                         thread-block size must
+                                                         be provided, there is
+                                                         no default provided.
+ cuda/hip_launch_t                         launch        Launches a device kernel,
+                                                         any code expressed within
+                                                         the lambda is executed
+                                                         on the device.
+ cuda/hip_thread_x_direct                  kernel (For)  Map loop iterates
+                                           launch (loop) directly to GPU threads
+                                                         in x-dimension, one
+                                                         iterate per thread
+                                                         (see note below about
+                                                         limitations)
+ cuda/hip_thread_y_direct                  kernel (For)  Same as above, but map
+                                           launch (loop) to threads in y-dim
+ cuda/hip_thread_z_direct                  kernel (For)  Same as above, but map
+                                           launch (loop) to threads in z-dim
+ cuda/hip_thread_x_loop                    kernel (For)  Similar to
+                                           launch (loop) thread-x-direct
+                                                         policy, but use a
+                                                         block-stride loop which
+                                                         doesn't limit number of
+                                                         loop iterates, this is
+                                                         not safe to use with
+                                                         Cuda/HipSyncThreads
```

Comment:
Should the part about thread safety be noted in a 'note' after the table for emphasis?

Reply:
I'll move it out and add something here to look at the notes.